tjdashboard/code/site/models/forms/dashboard.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<fieldset>
+		<field
+			name="dashboard_id"
+			type="text"
+			readonly="true"
+			class="readonly"
+			default="0"
+			description="JGLOBAL_FIELD_ID_DESC"/>
+
+		<field
+			name="asset_id"
+			type="hidden"
+		/>
+
+		<field
+			name="title" type="text"
+			filter="safehtml"
+			label="COM_TJDASHBOARD_FORM_LBL_DASHBOARD_TITLE"
+			description="COM_TJDASHBOARD_FORM_DESC_DASHBOARD_TITLE"
+			size="10" />
+
+		<field
+			name="alias" type="text"
```

Comment:
This should also change to string maybe

Reply:
@coolbung  Why we should use filters for alias/title.User should able to add whatever they want we need to take care at point of rendering that we should escaping the html tags. We can look at the Article xml file for reference. What you think ?

Reply:
Ok for title. But why do you need HTML in the alias field ?

Reply:
OK, Yeah alias should be string only as it will be in urls so there should not be tags there.

Reply:
OK